desktop/src/main/java/bisq/desktop/main/offer/MutableOfferDataModel.java

```diff
@@ -133,6 +133,9 @@
     protected boolean allowAmountUpdate = true;
     private final TradeStatisticsManager tradeStatisticsManager;
 
+    private final Predicate<ObjectProperty<Coin>> isPositiveAmount = (c) -> c.get() != null && !c.get().isZero();
+    private final Predicate<ObjectProperty<Price>> isPositivePrice = (p) -> p.get() != null && !p.get().isZero();
+    private final Predicate<ObjectProperty<Volume>> isPositiveVolume = (v) -> v.get() != null && !v.get().isZero();
```

Comment:
Shouldn't these be `isNonZero...`?

Reply:
Fixed in 2ae6bfa, sub branch [5-suggested-changes](https://github.com/ghubstan/bisq/tree/5-suggested-changes).  Fixes for this PR will go into a separate PR.